### PR TITLE
Fetch api

### PR DESCRIPTION
**Pull request by Bifabig**
In this PR:
- I used fetch API to generate random quotes.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -9,35 +10,70 @@ const Calculator = () => {
...
+        setHasError(true);
+      }
+      setIsLoading(false);
+    };
+    fetchData();
+  }, [setQuotes, setIsLoading]);
+
+  if (hasError) return <div>Something went wrong!</div>;
+
+  if (isLoading) return <div>Loading...</div>;
```
Comment (gemmen29) [inline src/components/Calculator.js]: - [ ] Kindly move the logic for fetching quote data in the new component (Quotes.js) as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_fetch_data_from_api.md#:~:text=Fetch%20data%20inside%20of%20a%20new%20component).

File: src/components/Calculator.js [hunk 2]
```diff
@@ -9,35 +10,70 @@ const Calculator = () => {
...
-          '2',
-          '3',
-          '+',
-          '0',
-          '.',
-          '=',
-        ]}
-      />
+      <div className="quote">
+        <Quotes quotes={quotes} />
```
Comment (gemmen29) [inline src/components/Calculator.js]: - [ ] Kindly import the new component (Quotes.js) into the main component (App.js) next to the Calculator component as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_fetch_data_from_api.md#:~:text=Import%20the%20new%20component%20into%20your%20main%20component%20(next%20to%20the%20Calculator)).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Bifabig, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @Bifabig 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**